integration/unplugin/examples/rollup/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "build": "npx rollup --config rollup.config.js"
+  }
```

Comment:
Shouldn't all the example `package.json`s have a dependency of `@danielx/civet`? Or are we waiting for a version that actually has the unplugin?

Actually, currently when I try to build an example I get e.g.
`Error: Cannot find module 'C:\Users\edemaine\Projects\Civet\node_modules\@danielx\civet\vite'`
(despite having Civet `npm link`ed)
I think it's finding the older Civet that we depend on for building. So this would be fixed once we release an unplugin version, so maybe worry about this later.

I did get the examples to build via `npm link @danielx/civet`. (Well, I tested only Vite and Astro.)

Reply:
They probably should be having `@danielx/civet` as a dependency. Without doing npm link in that specific folder currently it traverses up to the node_modules that do have `@danielx/civet`, which happens to link to older version.